ra/ra.go

```diff
@@ -720,7 +720,7 @@ func (ra *RegistrationAuthorityImpl) checkCertificatesPerNameLimit(ctx context.C
 		count, ok := counts[name]
 		if !ok {
 			// Shouldn't happen, but let's be careful anyhow.
-			return errors.New("StorageAuthority failed to return a count for every name")
+			return errors.New("CountCertificatesByNames failed to return a count for every name")
```

Comment:
I actually think that returning `probs.ServerInternal` from RPCs (as opposed to making a `probs.ServerInternal` inside the WFE) is generally incorrect. All errors that aren't of specific, recognized types will get turned into `ServerInternal` on rendering by the WFE.